cps/static/js/table.js

```diff
@@ -93,6 +93,116 @@ $(function() {
+        var pathname = document.getElementsByTagName("script"), src = pathname[pathname.length-1].src;
+        var path = src.substring(0,src.lastIndexOf("/"));
+        $("#restrict-elements-table").bootstrapTable({
+            formatNoMatches: function () {
+                return "";
+            },
+            url: path + "/../../ajax/listrestriction/" + type,
+            rowStyle: function(row, index) {
+                console.log('Reihe :' + row + ' Index :'+ index);
+                if (row.id.charAt(0) == 'a') {
+                    return {classes: 'bg-primary'}
+                }
+                else {
+                    return {classes: 'bg-dark-danger'}
+                }
+            },
+            onClickCell: function (field, value, row, $element) {
+                if(field == 3){
+                    console.log("element")
+                    $.ajax ({
+                        type: 'Post',
+                        data: 'id=' + row.id + '&type=' + row.type + "&Element=" + row.Element,
+                        url: path + "/../../ajax/deleterestriction/" + type,
+                        async: true,
+                        timeout: 900,
+                        success:function(data) {
+                            $.ajax({
+                                method:"get",
+                                url: path + "/../../ajax/listrestriction/"+type,
+                                async: true,
+                                timeout: 900,
+                                success:function(data) {
+                                    $("#restrict-elements-table").bootstrapTable("load", data);
+                                }
+                            });
+                        }
+                    });
+                }
+            },
+            striped: false
+        });
+        $("#restrict-elements-table").removeClass('table-hover');
+        $("#restrict-elements-table").on('editable-save.bs.table', function (e, field, row, old, $el) {
+            console.log("Hallo");
+            $.ajax({
+                url: path + "/../../ajax/editrestriction/"+type,
+                type: 'Post',
+                data: row //$(this).closest("form").serialize() + "&" + $(this)[0].name + "=",
+            });
+        });
+        $("[id^=submit_]").click(function(event) {
+            // event.stopPropagation();
+            // event.preventDefault();
+            $(this)[0].blur();
+            console.log($(this)[0].name);
+            $.ajax({
+                url: path + "/../../ajax/addrestriction/"+type,
+                type: 'Post',
+                data: $(this).closest("form").serialize() + "&" + $(this)[0].name + "=",
+                success: function () {
+                $.ajax ({
+                    method:"get",
+                    url: path + "/../../ajax/listrestriction/"+type,
+                    async: true,
+                    timeout: 900,
+                    success:function(data) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 24 spaces but found 20.](https://app.codacy.com/manual/Ozzie-IsaacsOrganization/calibre-web/pullRequest?prid=4846308)